test/definitions/ohi/linux/elasticsearch-debian.json

```diff
@@ -42,16 +42,6 @@
                     "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infrastructure/ohi/elasticsearch/debian.yml"
                 ]
             }
-        },
-        {
-            "id": "nr_infra_is_having_data",
-            "resource_ids": ["host1"],
-            "provider": "newrelic",
-            "source_repository": "https://github.com/newrelic/open-install-library.git",
-            "deploy_script_path": "test/deploy/assertions/recipe-is-valid/roles",
-            "params": {
-                "recipe_validation_nrql_url": "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infrastructure/ohi/elasticsearch/debian.yml"
-            }
```

Comment:
Just curious - why did this get removed in all the test definition files?

Reply:
This is a second validation after running the CLI. It's a bit optional, and I've hesitated to remove it because it *should* work after the CLI has already validated.
For some reasons, when running in the context of a PR, the test sometime failed on that validation. I've been removing it slowly on some OHIs to reduce the false positive

Reply:
Ah gotcha, makes sense